buckets.js

```diff
@@ -27,7 +27,8 @@ define('buckets', [], function() {
             // Camel-case it.
             property = property[0].toUpperCase() + property.substr(1);
 
-            for (var i = 0, e; e = prefixes[i++];) {
+            for (var i = 0, e; i < prefixes.length; i++) {
```

Comment:
I [made a JSPerf](http://jsperf.com/length-in-for-loop-big) to see if it is slower checking the length and it doesn't appear to be. Interestingly the form that was here before that doesn't check length at all appears to be significantly slower (and is vulnerable to subtle unexpected falsyness bugs).

I'll move the `var` to the inside of the loop.